torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1714,19 +1714,19 @@ def test_rref_get_future(self):
             rref = rpc.remote(worker_name(1), torch.add, args=(1, 1))
             rref.to_here()
             fut = rref._get_future()
-            self.assertIsInstance(fut, torch.distributed.rpc.Future)
+            self.assertIsInstance(fut, torch._C.Future)
```

Comment:
What's the difference between `torch.jit.Future` and `torch._C.Future`?

Reply:
`torch.jit.Future` is alias to `torch._jit_internal.Future`, which is a typing class, not the actual implementation.

`torch._C.Future` is the implementation, `PythonFutureWrapper`.